swap/common_test.go

```diff
@@ -99,12 +99,12 @@ func newBaseTestSwapWithParams(t *testing.T, key *ecdsa.PrivateKey, params *Para
 	}
 	log.Debug("creating simulated backend")
 	owner := createOwner(key)
-	swapLog = newSwapLogger(params.LogPath, params.BaseAddrs)
+	swapLog := newSwapLogger(params.LogPath, params.BaseAddrs.Address())
```

Comment:
also: can i ask why `Address` is being called now?

Reply:
fixed `swapLog` to `swapLogger`.
I refactored it to use the []byte for the overlay network address in logging.
But when I changed this I did not use the underlying address, so I should leave it as it was with BaseAddrs and log using ShortString

Here is an excerpt of `ShortString`
```
swarm/network/network.go

func (a *BzzAddr) ShortString() string {
	return fmt.Sprintf("%s:%s", a.ShortOver(), a.ShortUnder())
}

// ShortOver returns shortened version of Overlay address
// It can be used for logging
func (a *BzzAddr) ShortOver() string {
....
}

// ShortUnder returns shortened version of Underlay address
// It can be used for logging
func (a *BzzAddr) ShortUnder() string {
	....
}

```